### DISCUSSION TITLE
Does global mutate with work useInfiniteSWR?

### DISCUSSION THREAD

**Discussion by ymc9** [vercel/swr#2843] [Q&A]
Hi guys, I read it here that I need to compute the key differently, however still no refetch is triggered:
https://swr.vercel.app/docs/pagination#global-mutate-with-useswrinfinite

After some debugging I found the refetch depends on a "_i" context flag which seems only set by bound mutate. 

https://github.com/vercel/swr/blob/1d8110900d1aee3747199bfb377b149b7ff6848e/infinite/src/index.ts#L181

![image](https://github.com/vercel/swr/assets/104139426/f4b62073-6167-4115-a6fd-eeeb5839179d)
![image](https://github.com/vercel/swr/assets/104139426/d191f256-c8e3-43f2-8f64-947ecdb5a035)

Am I understanding it correctly? Anything else I need to do to get global mutate to work? My goals is to mutate without data (just refetch).

Thanks!

**Comment by koba04** [comment]
Could you create a CodeSandbox project to reproduce your issue?
